### PR TITLE
Add Coyote

### PR DESCRIPTION
**Pull request by thecoder08**
## Adding a language

- [x] The code displays "Hello World"
- [x] I have updated the readme to include the new language
- [x] I have incremented the language count in the readme

#### Link to programming language: 
https://github.com/thecoder08/coyote

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -8,8 +8,7 @@ Thanks to everyone who continues to contribute; new languages are created every
 Make sure to see [contributing.md](/contributing.md) for instructions on contributing to the project!
 
 <!--Languages start-->
-
```
Comment (Richienb) [inline readme.md:11]: There shouldn't be an extra newline here

Comment (thecoder08) [reply readme.md:11]: ah sorry, not sure how that happened

**Comment by MrBrain295** [comment]
This doesn’t comply with our conflict of interest policy.

**Comment by thecoder08** [comment]
So @MrBrain295  that means I need the language to become more popular and let someone else add it that isn't asscociated with me?

**Comment by MrBrain295** [comment]
> So @MrBrain295 that means I need the language to become more popular and let someone else add it that isn't asscociated with me?
Yes.

**Pull request closed without merging**